.github/workflows/review.yml

```diff
@@ -50,12 +50,12 @@ jobs:
           K8S_SECRET_SENTRY_ENVIRONMENT: "test"
           K8S_SECRET_TOKEN_AUTH_ACCEPTED_AUDIENCE: "https://api.hel.fi/auth/helsinkiprofile"
           K8S_SECRET_TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX: "helsinkiprofile"
-          K8S_SECRET_TOKEN_AUTH_AUTHSERVER_URL: "https://api.hel.fi/sso-test/openid"
+          K8S_SECRET_TOKEN_AUTH_AUTHSERVER_URL: "https://tunnistamo.${{ env.BASE_DOMAIN }}/openid"
           K8S_SECRET_TOKEN_AUTH_REQUIRE_SCOPE: 1
           K8S_SECRET_OIDC_CLIENT_ID: ${{ secrets.GH_QA_OIDC_CLIENT_ID }}
           K8S_SECRET_OIDC_CLIENT_SECRET: ${{ secrets.GH_QA_OIDC_CLIENT_SECRET }}
-          K8S_SECRET_TUNNISTAMO_API_TOKENS_URL: "https://api.hel.fi/sso-test/api-tokens"
-          K8S_SECRET_GDPR_AUTH_CALLBACK_URL: "https://profiili.test.kuva.hel.ninja/gdpr-callback"
+          K8S_SECRET_TUNNISTAMO_API_TOKENS_URL: "https://tunnistamo.${{ env.BASE_DOMAIN }}/api-tokens"
+          K8S_SECRET_GDPR_AUTH_CALLBACK_URL: "https://profiili..${{ env.BASE_DOMAIN }}/gdpr-callback"
```

Comment:
Is there an extra period here `profiili..`?

Reply:
Yes, fixed. Thank you.